.github/workflows/atomistics-compat.yml

```diff
@@ -23,24 +23,34 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: pyiron_atomistic
+      run: |
+        git clone https://github.com/pyiron/pyiron_atomistics ../pyiron_atomistics
+        grep -v "pyiron_base" ../pyiron_atomistics/.ci_support/environment.yml > environment.yml
+        tail -n +4 .ci_support/environment.yml >> environment.yml
```

Comment:
What happens if we update a dependency in base but not (yet) in atomistics? would this work or throw an error due to twice the same dependency with different versions?

Reply:
Here is an example [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/jan-janssen/multi-def-conda/HEAD)

Reply:
I wrote a small python script to merge the environments https://github.com/pyiron/pyiron_base/blob/compatfix/.ci_support/condamerge.py